templates/zerver/help/restrict-account-creation.md

```diff
@@ -51,8 +51,10 @@ Regardless of whether invitations are required, you can:
 
 {!owner-only.md!}
 
-You can restrict the ability to invite new users to join your
-Zulip organization to specific [roles](/help/roles-and-permissions).
+You can restrict the ability to invite new users to join your Zulip organization
+to specific [roles](/help/roles-and-permissions). To protect your organization,
+while permission to send out individual invites is configurable, creating
```

Comment:
Do we want to say "individual email invitations" instead of "individual invites" to keep that language consistent between the two help center pages?